conf/machine/raspberrypi4-64.conf

```diff
@@ -0,0 +1,17 @@
+# Raspberry Pi profile
```

Comment:
Rename raspberrypi4-64 to raspberrypi4b-64.conf , then correct the related things accordingly.